packages/tocco-ui/src/Icon/Icon.js

```diff
@@ -40,9 +40,12 @@ class Icon extends React.Component {
   }
 
   render() {
-    const filteredProps = _omit(this.props, ['dense', 'onLoaded', 'position', 'theme'])
+    const icon = (typeof this.props.icon === 'string' && this.props.icon.includes(','))
```

Comment:
When prop.icon is defined correctly this code is not needed. May extend PropTypes validation by allowing only strings without comma and remove this code.

Reply:
I dont quite get your comment. That you can use a comma separated string is a feature. I think in most cases it's friendlier this way.